api-tests/config/conf.js

```diff
@@ -0,0 +1,13 @@
+global.expect = require('chai').expect
+global.request = require('supertest')
+global.HttpStatus = require('http-status-codes')
+
+module.exports = {
+    BASE_URL: process.env.BASE_URL,
+    reporter: 'node_modules/mochawesome',
+    'reporter-option': [
+        'overwrite=true',
+        'reportTitle=My\ Custom\ Title',
```

Comment:
Report title

Reply:
I will cover that section once the api tests are successfully run through the pipeline 👍